pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -210,10 +210,10 @@ func (i *Controller) processNextObject(objWorkqueue workqueue.RateLimitingInterf
 	return true
 }
 
-func (i *Controller) enqueueObject(obj interface{}, workqueue workqueue.RateLimitingInterface) {
+func (i *Controller) enqueueObject(obj interface{}, rateLim workqueue.RateLimitingInterface) {
```

Comment:
Why does it complain about the name `workqueue`? I prefer that as that's what it really is with rate limiting characteristic. Or rename to `queue` if that satisfies it.

Reply:
Because it shaddows the imported `k8s.io/client-go/util/workqueue`.

Reply:
Changed it to `queue` in the latest, thanks. :+1: